client/src/components/ListVideos.jsx

```diff
@@ -1,21 +1,23 @@
+import DeleteVideo from "./DeleteVideo";
 import React, { useEffect, useState } from "react";
 
 function ListVideos() {
 	const [videos, setVideos] = useState([]);
 
-	useEffect(() => {
-		const fetchVideos = async () => {
-			try {
-				const response = await fetch("/api/videos");
-				const data = await response.json();
-				setVideos(data.data);
-			} catch (error) {
-				console.error("Error fetching videos:", error);
-			}
-		};
+	const fetchVideos = async () => {
+		try {
+			const response = await fetch("/api/videos");
+			const data = await response.json();
+			setVideos(data.data);
+		} catch (error) {
+			console.error("Error fetching videos:", error);
+		}
+	};
 
+	useEffect(() => {
 		fetchVideos();
 	}, []);
```

Comment:
I liked this approach, calling the function rather than creating it inside the useEffect statement.